### DISCUSSION TITLE
posts/tutorial0/

### DISCUSSION THREAD

**Discussion by giscus** [7w1/7w1.github.io#1] [Announcements]
# posts/tutorial0/

Intro

https://7w1.github.io/posts/tutorial0/

**Comment by DanlarScott** [comment]
I was thinking about trying to play this masterpiece again, (I dropped it years ago because it was overwhelming..lot of stuff, not knowing where to start etc.) now I'm reading your tutorials and GOD is what I was looking for. A lot of video-tutorials are very "information-heavy", leaving your brain like a sieve >.< because they try to explain many things at once..

They don't have a video where they explain "step by step" how to get started etc without as I said at the beginning, overloading you.. With this tutorial I can learn the basics, step by step, at my own pace with the necessary information. So now thanks to you [edit] and Defran (and its new revised tutorial-series, thanks again for mentioning it at the beginning of the tutoria!  without it I would not have seen this videos) [final Edit]  I will try to get into this game for a (hopefully) long time. Greetings.

**Reply by 7w1** [reply]
Thanks for the kind feedback! I hope you manage to get into it. And I should probably get the the next part of this tutorial out soon...

**Reply by DanlarScott** [reply]
Thank you for all your hard work! take your time mate ^_^ (dont rush).  I have just started the first part of the tutorial so Im sure that I will need months to process everything explained here (and I won't even touch the tip of the iceberg) because I'm sure that this game have TONS of things left for me to discover and learn.  Oh man Im really exited! ^_^ thank you again.

**Comment by teqqra** [comment]
Really thansks for the tutorial!

**Comment by herendicristi** [comment]
when trying to download or visit any pentarch site it says the site is down, has it been abandoned?

**Reply by 7w1** [reply]
No, the site’s just down at the moment. Here’s a mirror for the game files: https://drive.google.com/drive/folders/15JJ0AGwqjkVSFEGPHltjJPiNoya7Dpy4
